README.md

```diff
@@ -20,12 +23,52 @@ Simply include AutoParcel in your project and make any of your `@AutoValue` anno
 
 Now build your project and enjoy your Parcelable Foo.
 
+## TypeAdapters
+
+Out of the box AutoValue: Parcel Extension support all of the types supported by the 
+[Parcel](https://developer.android.com/reference/android/os/Parcel.html) class, but sometimes you
+may need to parcel other types, like booleans or Dates.  You can do this using a custom TypeAdapter.
```

Comment:
booleans are already supported by this plugin and Date is serializable. Not sure what types would be useful examples... ArrayMap? SparseLongArray?